src/components/KTextArea/KTextArea.vue

```diff
@@ -18,16 +18,28 @@
       </template>
     </KLabel>
 
-    <textarea
-      :id="textAreaId"
-      v-bind="modifiedAttrs"
-      :aria-invalid="ariaInvalid"
-      class="input-textarea"
-      :class="[resizable || isResizable ? 'resizable' : undefined]"
-      :rows="rows"
-      :value="getValue()"
-      @input="inputHandler"
-    />
+    <div
+      class="input-wrapper"
+      :class="{
+        autosize,
+        legacy: !SUPPORT_FIELD_SIZING_CONTENT
+      }"
+      :data-value="SUPPORT_FIELD_SIZING_CONTENT ? null : currValue"
+    >
```

Comment:
question: does the `input-wrapper` class inherit styles from anywhere as it didn't exist before?

Reply:
Didn't notice any of that in the sandbox, I'll put up a preview PR in Konnect.